src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
@LaurentGoderre no. The [GCweb PR](https://github.com/wet-boew/GCWeb/pull/1237) that re-add the data-json plugin need that function. Also, that function is needed by PR #7983 which add a cache buster for data-ajax.

Let discuss at the next PR review meeting.

/cc @lucas-hay